stackslib/src/chainstate/nakamoto/mod.rs

```diff
@@ -287,6 +287,22 @@ pub static NAKAMOTO_CHAINSTATE_SCHEMA_6: &[&str] = &[
     "CREATE INDEX IF NOT EXISTS nakamoto_block_headers_by_ch_bv ON nakamoto_block_headers(consensus_hash, burn_view);"
 ];
 
+pub static NAKAMOTO_CHAINSTATE_SCHEMA_7: &[&str] = &[
+    r#"
+    UPDATE db_config SET version = "12";
+    "#,
+    // Add a `total_tenure_size` field to the block header row, so we can keep track
```

Comment:
You might want to add a comment here (a SQL comment, specifically, so it shows up when querying the schema) saying that the values of `total_tenure_size` _cannot_ be consensus-critical, since existing nodes which migrate to this schema version will report tenure sizes of 0 bytes whereas nodes that boot from genesis will report the true tenure size.